[2021-10-03 00:33:46,786] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_A 2021-10-03T00:33:45.208996+00:00 [queued]>
[2021-10-03 00:33:46,822] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_A 2021-10-03T00:33:45.208996+00:00 [queued]>
[2021-10-03 00:33:46,825] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-03 00:33:46,827] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-10-03 00:33:46,829] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-03 00:33:46,852] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): Training_Model_A> on 2021-10-03T00:33:45.208996+00:00
[2021-10-03 00:33:46,872] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'Training_Model_A', '2021-10-03T00:33:45.208996+00:00', '--job-id', '54', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp8ladf_9l', '--error-file', '/tmp/tmpn240ages']
[2021-10-03 00:33:46,874] {standard_task_runner.py:77} INFO - Job 54: Subtask Training_Model_A
[2021-10-03 00:33:46,864] {standard_task_runner.py:52} INFO - Started process 23979 to run task
[2021-10-03 00:33:47,008] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.Training_Model_A 2021-10-03T00:33:45.208996+00:00 [running]> on host d6b4131456ad
[2021-10-03 00:33:47,079] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=Training_Model_A
AIRFLOW_CTX_EXECUTION_DATE=2021-10-03T00:33:45.208996+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-03T00:33:45.208996+00:00
[2021-10-03 00:33:47,105] {python.py:151} INFO - Done. Returned value was: None
[2021-10-03 00:33:47,124] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=Training_Model_A, execution_date=20211003T003345, start_date=20211003T003346, end_date=20211003T003347
[2021-10-03 00:33:47,184] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-03 00:33:47,215] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
